### PR TITLE
Add PagesController and view templates for homepage, incomany, attend and sponsor

### PR DESCRIPTION
**Pull request by Tenicia**
### Homepage

schermafbeelding 2017-03-28 om 12 36 43

### CODE REVIEW

**Comment by Tenicia** [comment]
Styling is done later!

File: config/routes.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 Rails.application.routes.draw do
-
+  root :to => 'pages#homepage'
```
Comment (miriamtocino) [inline config/routes.rb]: We don't use hash rockets anymore :wink: that's old Ruby!

Please change into:

```ruby
root to: 'pages#homepage'
```

Also remove extra line beneath it, line 4.

**Comment by arnoFleming** [comment]
Did you create a ticket to add styling later?

File: config/routes.rb [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
 
 
   get "book_in_company/home", :to => "book_in_company#home"
```
Comment (miriamtocino) [inline config/routes.rb]: What about handling the different static pages for the public, incompany and sponsor pages from your `PagesController` as well?

If you do that, these 3 routes would be changed into this:

```ruby
  get '/public', to: 'pages#public'
  get '/incompany', to: 'pages#incompany'
  get '/sponsor', to: 'pages#sponsor'
```

And then in your `PagesController` you just add those methods:

```ruby
class PagesController < ApplicationController
  def homepage
  end  

  def public
  end  

  def incompany
  end  

  def sponsor
  end  
end
```

You just need to create corresponing view files.

**Comment by miriamtocino** [comment]
@Tenicia And don't forget to add corresponding labels to the pull!

File: app/views/testimonials/_form.html.erb [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+  </div>
+</div>
+
+ <img class="img-circle" width="200" height="200" src="https://www.codaisseur.com/assets/codaisseur-logo-9a3b4e4a4929449d09c6504c4d67d2dfc4abbed8d8f4333c8cf441dc63d49a11.svg">
+ <h3>What people say about TOC</h3>
+ <p>testimonials</p>
+</div>
+
+<section id="carousel">
+	<div class="container">
```
Comment (arnoFleming) [inline app/views/testimonials/_form.html.erb]: please fix your indenting. Use 2 spaces consistently in all files

In this file you use 0 spaces(line 6), 2 spaces, 1 space(line 13), tabs (line 19) and lots of spaces(line 25).

Your colleagues will frown upon you...

File: config/routes.rb [hunk 4]
```diff
@@ -1,7 +1,6 @@
 Rails.application.routes.draw do
   root to: 'pages#homepage'
 
-
```
Comment (arnoFleming) [inline config/routes.rb]: thanks!

File: app/assets/stylesheets/pages.scss [hunk 5]
```diff
@@ -0,0 +1,23 @@
+// Place all the styles related to the pages controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+.image_logo {
+    width: 20px;
+
```
Comment (arnoFleming) [inline app/assets/stylesheets/pages.scss]: please remove newline

File: app/views/pages/homepage.html.erb [hunk 6]
```diff
@@ -0,0 +1,16 @@
+<div class="text-center">
+    <div class="image_logo">
```
Comment (arnoFleming) [inline app/views/pages/homepage.html.erb]: please fix indenting here as well

File: app/views/testimonials/_form.html.erb [hunk 7]
```diff
@@ -0,0 +1,76 @@
+
+
+
```
Comment (arnoFleming) [inline app/views/testimonials/_form.html.erb]: please don't start a file with newlines

File: app/views/pages/homepage.html.erb (hunk 6 above)
Comment (Tenicia) [inline app/views/pages/homepage.html.erb]: Are you looking at commented Homepage (2)?
I removed the " image log class".

File: app/views/testimonials/_form.html.erb (hunk 7 above)
Comment (Tenicia) [inline app/views/testimonials/_form.html.erb]: My bad...changed! Homepage(3)

File: config/routes.rb (hunk 2 above)
Comment (Tenicia) [inline config/routes.rb]: Done in ' Homepage(3).

**Comment by Tenicia** [comment]
#23 
schermafbeelding 2017-03-28 om 15 55 37

File: app/views/testimonials/_form.html.erb [hunk 8]
```diff
@@ -0,0 +1,71 @@
...
+              <blockquote>
+                <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quidem, veritatis nulla eum laudantium totam tempore optio doloremque laboriosam quas, quos eaque molestias odio aut eius animi. Impedit temporibus nisi accusamus.</p>
+              </blockquote>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</section>
```
Comment (miriamtocino) [inline app/views/testimonials/_form.html.erb]: This file shouldn't be part of this pull. Please remove it :smile:

**Comment by Tenicia** [comment]
Tests added and run successfully.:

Finished in 0.0564 seconds (files took 3.54 seconds to load)
4 examples, 0 failures

require 'rails_helper'

RSpec.describe PagesController, type: :controller do

  describe "GET views PagesController" do

   it "gets the :homepage view" do
      get :homepage
      expect(response).to have_http_status(:success)
   end

   it "gets the :sponsor view" do
      get :sponsor
      expect(response).to have_http_status(:success)
   end

   it "gets the :incompany view" do
      get :incompany
      expect(response).to have_http_status(:success)
    end

   it "gets the :public view" do
      get :public
      expect(response).to have_http_status(:success)
   end
  end
end

File: config/routes.rb [hunk 9]
```diff
@@ -1,11 +1,12 @@
 Rails.application.routes.draw do
+  root to: 'pages#homepage'
 
   devise_for :users
 
-  get "book_in_company/home", :to => "book_in_company#home"
+  get "book_in_company/home", :to => "pages#incompany"
```
Comment (Cesurhan) [inline config/routes.rb]: Maybe these hash rockets need to be modified?

File: config/routes.rb [hunk 10]
```diff
@@ -1,11 +1,12 @@
...
   devise_for :users
 
-  get "book_in_company/home", :to => "book_in_company#home"
+  get "in_company", to:  "pages#incompany"
 
-  get "sponsor/home", :to => "sponsor#home"
+  get "sponsor", to:  "pages#sponsor"
 
-  get "attend/home", :to => "attend#home"
+  get "attend", to: "pages#attend"
```
Comment (miriamtocino) [inline config/routes.rb]: Remove empty space

**Pull request merged**